test/remote/gateways/remote_simetrik_test.rb

```diff
@@ -53,12 +53,10 @@ def setup
         id: rand(100000000000..999999999999).to_s,
         datetime_local_transaction: Time.new.strftime('%Y-%m-%dT%H:%M:%S.%L%:z'),
         description: 'apopsicle',
-        installments: 1,
-        amount: {
-          currency: 'USD',
-          vat: 19
-        }
```

Comment:
big improvement considering we don't want customers to have to pass a rich object for `amount` 👍